src/main/java/io/lettuce/core/AbstractRedisAsyncCommands.java

```diff
@@ -1582,6 +1582,11 @@ public RedisFuture<Long> publish(K channel, V message) {
         return dispatch(commandBuilder.publish(channel, message));
     }
 
+    @Override
```

Comment:
Typically, we sort methods by their name. It can be that some methods are out of order already.

Reply:
i think its nice to keep similar or related things together in general. 
But i am updating it to have them sorted to keep in harmony with the rest. thanks.